manager/integration/tests/test_stress.py

```diff
@@ -160,6 +166,14 @@ def delete_random_snapshot(longhorn_api_client, volume_name):
         purge_random_snapshot(longhorn_api_client, volume_name, snapshot_name)
 
 
+def remove_datafile(pod_name):
+    file_path = STRESS_RANDOM_DATA_DIR + STESS_DATAFILE_NAME_PREFIX + \
```

Comment:
Use `os.path.join` instead to prevent missing or additional `/` in the strings.

Reply:
done.